content/community/getting-involved/designing.html

```diff
@@ -5,26 +5,78 @@
 tags = []
 +++
 
-<p>A person with experience in graphic design has potential to be a very great help in ensuring that Haiku looks good and presents itself well to the public. The areas of design which a skilled hand is needed by an open source project are varied, but the style used needs to remain true to the over all branding of the project.</p> <p>Therefore there are a number of guides to help designers stay true to the over all Haiku theme.</p> <p>Check out the following areas where we need help with design:</p>
-<HR>
+<p>A person with experience in graphic design has potential to be a very great help in ensuring that Haiku looks good and presents itself well to the public. The areas of design which a skilled hand is needed by an open source project are varied, but the style used needs to remain true to the over all branding of the project.</p>
+
+<p>The Haiku project primarily needs <b>UX/UI designers</b>, <b>graphic designers</b> and all sorts of <b>visual artists</b>.
```

Comment:
Because I wasn't sure how not to reject other sort of artists, but I didn't do that, because I decided to include the umbrella term "visual artists". Should we include video editors?

Reply:
video things would be more "marketing and communication" than "design". Yet another place where people could contribute and there is already a team in place for that

Reply:
I will not modify this because adding redirects is currently out of scope.